packages/client/modules/userDashboard/containers/OrgBilling/OrgBillingRoot.tsx

```diff
@@ -20,18 +21,31 @@ interface Props {
 }
 
 const OrgBillingRoot = ({organization}: Props) => {
-  const atmosphere = useAtmosphere()
+  const queryRef = useQueryLoaderNow<OrgBillingRootQuery>(orgBillingRootQuery, {
+    orgId: organization.id,
+    first: 3
+  })
   return (
-    <QueryRenderer
-      environment={atmosphere}
-      query={query}
-      variables={{orgId: organization.id, first: 3}}
-      fetchPolicy={'store-or-network' as any}
-      render={renderQuery(OrgBilling, {props: {organization}, size: LoaderSize.PANEL})}
-    />
+    <Suspense fallback={''}>
+      {queryRef && <OrgBillingContainer queryRef={queryRef} organization={organization} />}
+    </Suspense>
   )
 }
 
+interface OrgBillingContainerProps {
+  queryRef: PreloadedQuery<OrgBillingRootQuery>
+  organization: OrgBillingRoot_organization
+}
+
+function OrgBillingContainer(props: OrgBillingContainerProps) {
+  const {queryRef, organization} = props
+  const data = usePreloadedQuery<OrgBillingRootQuery>(query, queryRef, {
+    UNSTABLE_renderPolicy: 'full'
+  })
+  const {viewer} = data
+  return <OrgBilling viewer={viewer} organization={organization} />
+}
+
 export default createFragmentContainer(OrgBillingRoot, {
```

Comment:
-1 If you touch this already, please convert the `createFragmentContainer` call to make it more readable overall